ci/pkl-pipelines/pay-dev/pr-ci.pkl

```diff
@@ -0,0 +1,248 @@
+amends "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.4#/Pipeline.pkl"
+import "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.4#/Pipeline.pkl"
+import "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.4#/TaskConfig.pkl"
+
+import "../common/pipeline_self_update.pkl"
+import "../common/shared_resources.pkl"
+import "../common/shared_resources_for_test_pipelines.pkl" as shared_test
+import "../common/PayResources.pkl"
+
+local e2e_test_apps = new Mapping<String, AppConfig> {
+  ["adminusers"] = new { tests = new { "card" } }
+  ["cardid"] = new { tests = new { "card" "zap" }; use_cardid_submodule = true }
+  ["connector"] = new { tests = new {"card" } }
+  ["endtoend"] = new { tests = new {"card" "products" "zap"} }
+  ["frontend"] = new { tests = new {"card" "products" "zap"} }
+  ["ledger"] = new { tests = new { "card" } }
+  ["products"] = new { tests = new { "products" } }
+  ["products_ui"] = new { tests = new { "products" }; node_build = true }
+  ["publicapi"] = new { tests = new {"card" "products" "zap"} }
+  ["publicauth"] = new { tests = new { "card" "zap" } }
+  ["selfservice"] = new { tests = new { "card" }; node_build = true }
+} // pay-ci-endtoend-config is different enough to break out
+
+local groupMap = new Mapping <String, Listing<String>> {
+  ["adminusers"] = new { "adminusers-e2e" }
+  ["cardid"] = new { "cardid-e2e" }
+  ["connector"] = new { "connector-e2e" }
```

Comment:
```suggestion
  ["connector"] = new { "card-connector-e2e" }
```

Reply:
JH and I decided to change this to make it the same as everything else.